feature/qos/otg_tests/egress_strict_priority_scheduler_with_bursty_traffic_test/README.md

```diff
@@ -0,0 +1,162 @@
+# DP-1.5: Egress Strict Priority scheduler with bursty traffic
+
+## Summary
+
+This test verifies the behavior of an egress strict priority scheduler under bursty traffic conditions. By configuring multiple priority queues with specific traffic classes and generating bursty traffic that exceeds the interface capacity in short durations, we will validate that the scheduler maintains strict priority order, prioritizing the transmission of higher-priority traffic even during bursts, potentially leading to drops in lower-priority traffic.
+
+## QoS traffic test setup:
+
+*   Topology:
+
+    *   2 input interfaces and 1 output interface with the same port speed. The
+        interface can be a physical interface or LACP bundle interface with the
+        same aggregated speed.
+
+    ```
+      ATE port 1
+          |
+         DUT--------ATE port 3
+          |
+      ATE port 2
+    ```
+
+*   Traffic classes:
+
+    *   We will use 6 traffic classes NC1, AF4, AF3, AF2, AF1 and BE1.
+
+*   Traffic types:
+
+    *   All the traffic tests apply to both IPv4 and IPv6 and also MPLS traffic.
+
+*   Queue types:
+
+    *   NC1/AF4/AF3/AF2/AF1/BE1 will have strict priority queues (be1 - priority 6, af1 - priority 5, ..., nc1 - priority 1)
+
+*   Test results should be independent of the location of interfaces. For
+    example, 2 input interfaces and output interface could be located on
+
+    *   Same ASIC-based forwarding engine
+    *   Different ASIC-based forwarding engine on same line card
+    *   Different ASIC-based forwarding engine on different line cards
+
+*   Test results should be the same for port speeds 100G and 400G.
+
+*   Counters should be also verified for each test case:
+
+    *   /qos/interfaces/interface/output/queues/queue/state/transmit-pkts
+    *   /qos/interfaces/interface/output/queues/queue/state/dropped-pkts
+    *   transmit-pkts should be equal to the number of Rx pkts on Ixia port
+    *   dropped-pkts should be equal to diff between the number of Tx and the
+        number Rx pkts on Ixia ports
+
+*   Latency:
+
+    *   Should be < 100000ns
+
+## Procedure
+
+* DUT Configuration:
```

Comment:
Please add headings formatted like DP-1.5.x as per the [template](https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#procedure)